server/cache.go

```diff
@@ -346,15 +346,15 @@ type clusterInfo struct {
 	regions *regionsInfo
 
 	activeRegions   int
-	writeStatistics *cache.LRU
+	writeStatistics cache.Cache
 }
 
 func newClusterInfo(id IDAllocator) *clusterInfo {
 	return &clusterInfo{
 		id:              id,
 		stores:          newStoresInfo(),
 		regions:         newRegionsInfo(),
-		writeStatistics: cache.NewLRU(writeStatLRUMaxLen),
+		writeStatistics: cache.NewDefaultCache(writeStatLRUMaxLen),
```

Comment:
maybe ARC more suitable for our `writerStatistics`, but we just need to peek the frequent cache t2. cc @disksing